std/process.d

```diff
@@ -908,6 +908,19 @@ unittest // Reopening the standard streams (issue 13258)
     assert(lines == ["foo", "bar"]);
 }
 
+version (Windows)
```

Comment:
Ah, I missed the fact that this is a Win32/64 inconsistency, I though it was a Win/POSIX inconsistency. Nevermind, then.